runltp-ng

```diff
@@ -141,10 +144,10 @@ if ($run) {
 	my %results;
 	$results{'sysinfo'} = utils::collect_sysinfo($backend);
 	my ($stats, $test_results) =
-	    utils::run_ltp($backend, $ltpdir, $run, $timeout,
-			   $include, $exclude);
+		utils::run_ltp($backend, $ltpdir, $run, $timeout,
+			$include, $exclude);
```

Comment:
Can we please avoid whitespace changes like this one?

The rest looks good to me.

Reply:
i really try. i have set up emacs to use whitespaces and there are things that apply automatically. sorry